src/locales/ko/pokemon-form.json

```
@@ -131,25 +178,55 @@
   "miniorViolet": "보라색 코어",
   "mimikyuDisguised": "둔갑한 모습",
   "mimikyuBusted": "들킨 모습",
+  "necrozmaDuskMane": "Dusk Mane",
+  "necrozmaDawnWings": "Dawn Wings",
+  "necrozmaUltra": "Ultra",
   "magearnaOriginal": "500년 전의 색",
   "marshadowZenith": "투지를 불태운 마샤도",
   "sinisteaPhony": "위작품",
   "sinisteaAntique": "진작품",
+  "milceryVanillaCream": "Vanilla Cream",
+  "milceryRubyCream": "Ruby Cream",
+  "milceryMatchaCream": "Matcha Cream",
+  "milceryMintCream": "Mint Cream",
+  "milceryLemonCream": "Lemon Cream",
+  "milcerySaltedCream": "Salted Cream",
+  "milceryRubySwirl": "Ruby Swirl",
+  "milceryCaramelSwirl": "Caramel Swirl",
+  "milceryRainbowSwirl": "Rainbow Swirl",
   "eiscueNoIce": "나이스페이스",
   "indeedeeMale": "수컷의 모습",
   "indeedeeFemale": "암컷의 모습",
   "morpekoFullBelly": "배부른 모양",
+  "morpekoHangry": "Hangry",
   "zacianHeroOfManyBattles": "역전의 용사",
+  "zacianCrowned": "Crowned",
   "zamazentaHeroOfManyBattles": "역전의 용사",
+  "zamazentaCrowned": "Crowned",
+  "kubfuSingleStrike": "Single Strike",
+  "kubfuRapidStrike": "Rapid Strike",
   "zarudeDada": "아빠",
+  "calyrexIce": "Ice Rider",
+  "calyrexShadow": "Shadow Rider",
+  "basculinMale": "Male",
+  "basculinFemale": "Female",
   "enamorusIncarnate": "화신폼",
+  "enamorusTherian": "Therian",
+  "lechonkMale": "Male",
+  "lechonkFemale": "Female",
+  "tandemausFour": "Family of Four",
+  "tandemausThree": "Family of Three",
   "squawkabillyGreenPlumage": "그린 페더",
   "squawkabillyBluePlumage": "블루 페더",
   "squawkabillyYellowPlumage": "옐로 페더",
   "squawkabillyWhitePlumage": "화이트 페더",
+  "finizenZero": "Zero",
+  "finizenZero": "Hero",
```

```suggestion
  "finizenZero": "Zero",
  "finizenZero": "Hero",
```

@Adri1, Please check duplicate key.